applications/ExternalSolversApplication/custom_utilities/feast_condition_number_utility.h

```diff
@@ -0,0 +1,250 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
```

Comment:
this file is about FEAST. It should not be in your application. Even more, it should be a part of a different commit